build/python.props

```diff
@@ -1,5 +1,6 @@
 <Project>
-  <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Functions.PythonWorker" Version="4.34.0" />
-  </ItemGroup>
+  <!-- See /eng/build/Workers.Python.props -->
+  <Target Name="_DoNotImport" BeforeTargets="Build">
+    <Error Text="This file should not be imported. Import '$(TargetsRoot)Workers.props'." />
```

Comment:
Should this be `Import '$(WorkersProps)'.`? Based off of what's currently in the dev branch

Reply:
That change will come in a follow up PR to port from dev.